oh-my-zsh.sh

```diff
@@ -84,12 +87,11 @@ if (( $zcompdump_refresh )); then
   # See https://github.com/ohmyzsh/ohmyzsh/commit/dd1a7269#commitcomment-39003489
   tee -a "$ZSH_COMPDUMP" &>/dev/null <<EOF
 
-$zcompdump_revision
-$zcompdump_fpath
+$zcompdump_meta
 EOF
```

Comment:
We could also get rid of the `tee` call, since "$ZSH_COMPDUMP" isn't derived from `$SHORT_HOST` anymore (see https://github.com/ohmyzsh/ohmyzsh/commit/dd1a7269#commitcomment-39003489). So a simple `echo` would suffice:
```zsh
echo "\n$zcompdump_meta" >>| "$ZSH_COMPDUMP"
```